docs/source/guides/worklock_guide.rst

```diff
@@ -7,29 +7,301 @@ WorkLock Guide
 Overview
 --------
 
-`WorkLock` is a novel, permissionless token distribution mechanism, developed at NuCypher, which requires participants to stake ETH and maintain NuCypher nodes in order to receive NU tokens.
+`WorkLock` is a novel, permissionless token distribution mechanism, developed at NuCypher, which requires participants
+to stake ETH and maintain NuCypher nodes in order to receive NU tokens.
 
-WorkLock offers specific advantages over ICO or airdrop as a distribution mechanism, chiefly: it selects for participants who are most likely to strengthen the network because they commit to staking and running nodes.
+WorkLock offers specific advantages over ICO or airdrop as a distribution mechanism, chiefly: it selects for participants
+who are most likely to strengthen the network because they commit to staking and running nodes.
 
-The WorkLock begins with an open bidding period, during which anyone seeking to participate can send ETH to the WorkLock contract to be escrowed on-chain.
+The WorkLock begins with an open bidding or `contribution` period, during which anyone seeking to participate can send
+ETH to the WorkLock contract to be escrowed on-chain.
 At any time, WorkLock participants can cancel their bid to forgo NU and recoup their escrowed ETH immediately.
```

Comment:
```suggestion
At any time during the contribution period, WorkLock participants can cancel their bid to forgo NU and recoup their escrowed ETH immediately.
```

Reply:
I guess during the contribution period and cancellation period ...? -> contribution period is a subset of the cancellation period.

Reply:
^ Ignore - read it without context

✔️